[CAS-1448] Extract markdown support in separate module (#2786)
GetStream/stream-chat-android: docusaurus/docs/Android/03-ui/03-chatui.mdx

}
```

### Transforming Message Text

You can easily provide a transformer that can transform and apply the message text to a given `TextView`. You need to override `ChatUI.messageTextTransformer` to an instance of `ChatMessageTextTransformer`s implementation.

```kotlin
ChatUI.messageTextTransformer = ChatMessageTextTransformer { textView: TextView, messageItem: MessageItem ->
    // Transform messages to upper case.
    textView.text = messageItem.message.text.uppercase()
}
```

> Stream UI TextView components doesn't have `android:autoLink` property set because it conflicts with Markdown plugins.

Reviewer: ```suggestion
Stream UI TextView components don't have `android:autoLink` property set because it conflicts with Markdown plugins.
```

> If you want to use a custom `ChatMessageTextTransformer` with text autolink, you can use `AutoLinkableTextTransformer`. After applying the transformation, it also makes the links clickable.

Reviewer: ```suggestion
:::note
You can use `AutoLinkableTextTransformer` if you want to apply custom transformation but keep links clickable.
:::
```
